drivers/misc/qseecom.c

```diff
@@ -3525,6 +3540,22 @@ static int __qseecom_send_cmd(struct qseecom_dev_handle *data,
 		pr_err("cache operation failed %d\n", ret2);
 		return ret2;
 	}
+
+	if (sb != NULL && rb != NULL && req->resp_len >= 0x31 + 16) {
+		if (rb[0] == 0) {
+			if (strncmp((uint8_t *)rb + 0x31,
+				    "HWC_Yoshino_Com_", 16) == 0)
+			{
+				((uint8_t *)rb)[0x30] = 1;
+				// 0=not_allowed, 1=locked, 2=unlocked,
+				// 3=allowed_when_sl_is_unlocked,
+				// 4=allowed_since_sl_is_unlocked,
+				// 5=unsupported_bl_status->generic error
+				//   (no info in security test screen "none")
+			}
+		}
```

Comment:
```suggestion
		if (rb[0] == 0 && strncmp((uint8_t *)rb + 0x31, "HWC_Yoshino_Com_", 16) == 0) {
			((uint8_t *)rb)[0x30] = 1;
			// 0=not_allowed, 1=locked, 2=unlocked,
			// 3=allowed_when_sl_is_unlocked,
			// 4=allowed_since_sl_is_unlocked,
			// 5=unsupported_bl_status->generic error
			//   (no info in security test screen "none")
		}
```

Reply:
This seems to be just formatting change - not really acceptable in my opinion.
I believe it is better to keep code lines shorter, less than 80 chars - is not that even required in kernel coding style to have the lines with less than 80 columns?

Reply:
I mainly combined the nested if to an AND to reduce the nesting level which is a readability improvement and makes the check(s) explicit.
No idea about the 80 chars TBH, especially as TABs are used I find that hard to reason about...

Reply:
Yes, seen the combined conditions with the 'and'. Still in the end the difference is only at formatting level.
Which is wrong in your case - it is very simple: tabs are always considered 8 chars wide in kernel code and there is 80 columns limit per line.
See the kernel coding style, for example here:
https://www.kernel.org/doc/html/v4.10/process/coding-style.html